docs/azure/sdk/snippets/dependency-injection/WebApplicationBuilder/Program.cs

```diff
@@ -1,18 +1,49 @@
 ﻿#region snippet_WebApplicationBuilder
 using Azure.Identity;
+using Azure.Messaging.ServiceBus;
+using Azure.Messaging.ServiceBus.Administration;
 using Microsoft.Extensions.Azure;
 
 WebApplicationBuilder builder = WebApplication.CreateBuilder(args);
 
+List<string> queueNames = await GetQueueNames();
+
 builder.Services.AddAzureClients(clientBuilder =>
 {
+    // Register clients for each service
     clientBuilder.AddSecretClient(new Uri("<key_vault_url>"));
     clientBuilder.AddBlobServiceClient(new Uri("<storage_url>"));
+    clientBuilder.AddServiceBusClientWithNamespace("<your_namespace>.servicebus.windows.net");
     clientBuilder.UseCredential(new DefaultAzureCredential());
+
+    // Register a subclient for each Service Bus Queue
+    foreach (string queue in queueNames)
+    {
+        clientBuilder.AddClient<ServiceBusSender, ServiceBusClientOptions>(
+            (_, _, provider) => provider.GetService<ServiceBusClient>()!
```

Comment:
I think we should avoid the nullability operators.  That's going to confuse people not using nullable checks - which is a large slice of .NET developers.

Leaving one comment, but I'd strip it from all examples, everywhere.